<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mayo::MainWindow</class>
 <widget class="QMainWindow" name="Mayo::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mayo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stack_Main">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_MainHome">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_AppIcon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../mayo.qrc">:/images/appicon_128.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_MainHome">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;You can create a &lt;/span&gt;&lt;a href=&quot;NewDocument&quot;&gt;&lt;span style=&quot; font-size:10pt; text-decoration: underline; color:#0000ff;&quot;&gt;New Document&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;or &lt;/span&gt;&lt;a href=&quot;OpenDocuments&quot;&gt;&lt;span style=&quot; font-size:10pt; text-decoration: underline; color:#0000ff;&quot;&gt;Open Existing Documents&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_MainControl">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter_Main">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="widget_Left" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_LeftContents" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="widget_LeftHeader" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QComboBox" name="combo_LeftContents">
                    <item>
                     <property name="text">
                      <string>Application tree</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Opened documents</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>File system</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QStackedWidget" name="stack_LeftContents">
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="page_ApplicationTree">
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QSplitter" name="splitter_ApplicationTree">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <widget class="WidgetApplicationTree" name="widget_ApplicationTree" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>0</horstretch>
                        <verstretch>1</verstretch>
                       </sizepolicy>
                      </property>
                     </widget>
                     <widget class="WidgetDocumentItemProps" name="widget_DocumentProps" native="true"/>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="page_OpenedDocuments">
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QListView" name="listView_OpenedDocuments">
                     <property name="horizontalScrollBarPolicy">
                      <enum>Qt::ScrollBarAlwaysOff</enum>
                     </property>
                     <property name="editTriggers">
                      <set>QAbstractItemView::NoEditTriggers</set>
                     </property>
                     <property name="textElideMode">
                      <enum>Qt::ElideMiddle</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="page_FileSystem">
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QTreeView" name="treeView_FileSystem">
                     <attribute name="headerVisible">
                      <bool>false</bool>
                     </attribute>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_LeftRightSeparator">
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget_Right" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_ControlGuiDocuments" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>1</number>
               </property>
               <property name="leftMargin">
                <number>2</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="ButtonFlat" name="btn_PreviousGuiDocument">
                 <property name="icon">
                  <iconset resource="../mayo.qrc">
                   <normaloff>:/images/previous.png</normaloff>:/images/previous.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>12</width>
                   <height>12</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="ButtonFlat" name="btn_NextGuiDocument">
                 <property name="icon">
                  <iconset resource="../mayo.qrc">
                   <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>12</width>
                   <height>12</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="combo_GuiDocuments">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="sizeAdjustPolicy">
                  <enum>QComboBox::AdjustToContents</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="line">
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="ButtonFlat" name="btn_CloseGuiDocument">
                 <property name="icon">
                  <iconset resource="../mayo.qrc">
                   <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>12</width>
                   <height>12</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="line_2">
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_Spacer" native="true">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_MouseCoords" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_PosX">
                      <property name="text">
                       <string>X=</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_ValuePosX">
                      <property name="text">
                       <string>?</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_PosY">
                      <property name="text">
                       <string>Y=</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_ValuePosY">
                      <property name="text">
                       <string>?</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_PosZ">
                      <property name="text">
                       <string>Z=</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_ValuePosZ">
                      <property name="text">
                       <string>?</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QStackedWidget" name="stack_GuiDocuments">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>2</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewDoc"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExportSelectedItems"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionReportBug"/>
    <addaction name="separator"/>
    <addaction name="actionAboutMayo"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionSaveImageView"/>
    <addaction name="actionInspectXDE"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionPreviousDoc"/>
    <addaction name="actionNextDoc"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Options"/>
   <addaction name="menuWindow"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionNewDoc">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAboutMayo">
   <property name="text">
    <string>About Mayo</string>
   </property>
  </action>
  <action name="actionReportBug">
   <property name="text">
    <string>Report Bug</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionSaveImageView">
   <property name="icon">
    <iconset resource="../mayo.qrc">
     <normaloff>:/images/camera_32.png</normaloff>:/images/camera_32.png</iconset>
   </property>
   <property name="text">
    <string>Save View to Image</string>
   </property>
  </action>
  <action name="actionExportSelectedItems">
   <property name="text">
    <string>Export selected items</string>
   </property>
  </action>
  <action name="actionInspectXDE">
   <property name="text">
    <string>Inspect XDE</string>
   </property>
  </action>
  <action name="actionPreviousDoc">
   <property name="icon">
    <iconset resource="../mayo.qrc">
     <normaloff>:/images/previous.png</normaloff>:/images/previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous Document</string>
   </property>
   <property name="toolTip">
    <string>Previous Document</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="actionNextDoc">
   <property name="icon">
    <iconset resource="../mayo.qrc">
     <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
   </property>
   <property name="text">
    <string>Next Document</string>
   </property>
   <property name="toolTip">
    <string>Next Document</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="actionCloseDoc">
   <property name="icon">
    <iconset resource="../mayo.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
   <property name="text">
    <string>Close Document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WidgetApplicationTree</class>
   <extends>QWidget</extends>
   <header>widget_application_tree.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetDocumentItemProps</class>
   <extends>QWidget</extends>
   <header>widget_document_item_props.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonFlat</class>
   <extends>QToolButton</extends>
   <header>button_flat.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../mayo.qrc"/>
 </resources>
 <connections/>
</ui>
